### DISCUSSION TITLE
Weekly Retraining

### DISCUSSION THREAD

**Discussion by wbond** [github-early-access/copilot-custom-models-beta#7] [Q&A]
We currently haven't rolled our custom model out to any significant usage over the past week since we started the training.

Should we plan on retraining every week, no matter what, during the beta phase to get any improvements the GitHub team may have made since our last training?

**Comment by justincampbell** [comment]
@wbond There are not likely to be large changes that would impact the model week-to-week. If you have a large amount of new source code/projects to add to the training set, that would be a great time to retrain, otherwise ~monthly is a good cadence now to pick up training improvements from our side. We're also discussing internally how to handle this. Thanks for raising the question!
